app/controllers/filters_controller.rb

```diff
@@ -67,14 +67,6 @@ def find_role
     @role = Role.find_by_id(role_id)
   end
 
-  def resource_base
```

Comment:
For future reference: This doesn't need the conditional anymore and thus is same as parent.

Reply:
This was actually already the case for a while, this change only made it clear.